inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -637,7 +637,8 @@ def __init__(self, *args, **kwargs):
 
         degree_type_schema = load_schema('elements/degree_type.json')
         degree_choices = [
-            (val, val.capitalize()) for val in degree_type_schema['enum']
+            (val, val.capitalize()) if val != 'phd' else ('phd', 'PhD')
+            for val in degree_type_schema['enum']
```

Comment:
This constructor starts to do too much.

These lines could be actually factored out into a small testable function:
```
def make_degree_choices():
```

The test would just check against the current expected output.